.github/workflows/cypress-tests.yml

```diff
@@ -0,0 +1,39 @@
+name: "dhis2: cypress tests"
+
+on: [push]
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        containers: [1, 2, 3]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        with:
+          record: true
+          parallel: true
+          group: e2e-chrome-parallel
+          browser: chrome
+          start: 'yarn start'
+          wait-on: 'http://localhost:3000'
+          # wait for 200 secs for the server to respond
+          wait-on-timeout: 200
+
+      - name: Upload artifacts
+        if: failure()
+        uses: actions/upload-artifact@v1
+        with:
+          name: failed-screenshots
+          path: cypress/screenshots
+    env:
+      CI: true
+      CYPRESS_RECORD_KEY: '6b0bce0d-a4e8-417b-bbee-9157cbe9a999'
+      REACT_APP_DHIS2_BASE_URL: 'https://debug.dhis2.org/ca-test-2.34'
```

Comment:
this instance I have created. Is an instance that simulates the latest of the version we are having with the seeded data our databases usually have at this time.

in https://debug.dhis2.org/ you can see the instance. I have left a message there so that people dont mess with the data.

One problem with this approach is that if the data from the database are changed it might cause our tests to fail. In these cases which should be rare we can spin up a new instance with all the data reseted.